### PR TITLE
update crds in make update

### PR DESCRIPTION
**Pull request by 27149chen**
update code generation for crd changes in make update

### CODE REVIEW

File: hack/update-crds.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#
+
+# Run operator-sdk generate k8s and operator-sdk generate openapi to update code after crd changes.
+
+if ! [ -x "$(command -v operator-sdk)" ]; then
+  echo 'Error: operator-sdk is not installed.' >&2
+  exit 1
+fi
+
+echo "run operator-sdk generate k8s"
```
Comment (oriyarde) [inline hack/update-crds.sh]: consider using "set -x" instead

Comment (oriyarde) [reply hack/update-crds.sh]: by "instead" i meant that there is no need for "echo"s

Comment (27149chen) [reply hack/update-crds.sh]: can you explain why?

Comment (oriyarde) [reply hack/update-crds.sh]: since "-x" already echoes the commands

Comment (27149chen) [reply hack/update-crds.sh]: ok, got you.

**Pull request merged**